client/packages/programs/src/api/hooks/document/useEncounterRegistriesByPrograms.ts

```diff
@@ -32,7 +35,9 @@ export const useEncounterRegistriesByPrograms = (
           result.nodes
             .map(encounter => {
               const program = programs.find(
-                p => p.document?.documentRegistry?.id === encounter.parentId
+                p =>
+                  p.document?.documentRegistry?.id === encounter.parentId &&
+                  p.status === ProgramEnrolmentNodeStatus.Active
```

Comment:
I would prefer keep the api hooks more general purpose. For example, it could be very confusing if you are looking for all encounters because its not clear from the hook name that it filters by active programs... It might be easier to jut filter the list of input `programs` before calling this hook, then its very clear on the caller side what you want to fetch...